source/minimessage/format.rst

```diff
@@ -0,0 +1,272 @@
+.. _minimessage-format:
+
+Format
+======
+
+The Minimessage language uses tags. Everything you do will be defined with tags. Tags have a start tag and an end tag (the :mm:`<reset>` tag is an exception here).
+Start tags are mandatory (obviously), end tags aren't outside of ``strict`` mode.
+:mm:`<yellow>Hello <blue>World<yellow>!` and :mm:`<yellow>Hello <blue>World</blue>!` and even :mm:`<yellow>Hello </yellow><blue>World</blue><yellow>!</yellow>` all do the same.
+
+All tag names are case-insensitive to reduce the possibility for conflict, but we recommend keeping all tag names lowercase (or at the very least, being consistent).
+
+Some tags have argument. Those look like this: :mm:`<tag:argument>stuff</tag>`. For example: :mm:`<hover:show_text:"<red>test:TEST">TEST` or :mm:`<click:run_command:test>TEST`
+As you can see, those sometimes contain components, sometimes just numbers, strings, or other types. Refer to the detailed docs below.
+
+Single (``'``) and double (``"``) quotes can be used interchangeably, but for your own sanity, please stay consistent, choose one for all your messages. MiniMessage *should* handle mismatched quotes nicely tho.
```

Comment:
Not sure if this is true any more

Reply:
me neither! what does 'nicely' even mean? probably best to remove the sentence lol